tests/unit/assessment/test_jobs.py

```diff
@@ -56,3 +57,77 @@ def test_job_crawler_with_no_owner_should_have_empty_creator_name():
     JobsCrawler(ws, sql_backend, "ucx").snapshot()
     result = sql_backend.rows_written_for("hive_metastore.ucx.jobs", "append")
     assert result == [JobInfo(job_id="9001", job_name="No Tasks", creator=None, success=1, failures="[]")]
+
+
+@pytest.mark.parametrize(
+    "jobruns_ids,cluster_ids,run_ids,failures",
+    [
+        (
+            ['notebook_task'],
+            ['outdated-autoscale'],
+            '["123"]',
+            '[{"task_key": "notebook", "failures": ["not supported DBR: 9.3.x-cpu-ml-scala2.12"]}]',
```

Comment:
Failures has to be ARRRAY<STRING>, here you have more complicated structure. If you supply test parameters, have them as python structures, not serialized jsons.

```suggestion
            ["not supported DBR: 9.3.x-cpu-ml-scala2.12"],
```